src_python/habitat_sim/simulator.py

```diff
@@ -439,6 +439,13 @@ def _sensors(self) -> Dict[str, "Sensor"]:
         # TODO Deprecate and remove
         return self.__sensors[self._default_agent_id]
 
+    @property
+    def scene_bb(self) -> mn.Range3D:
+        """
+        The bounding box of the currently instantiated scene.
+        """
+        return self.get_active_scene_graph().get_root_node().cumulative_bb
+
```

Comment:
Instead of doing this in simulator.py, how about we put it in a sim binding? Or would that not work

Reply:
Yeah, we can do that. Did this for quick implementation initially.